lib/app.jsx

```diff
@@ -99,6 +99,11 @@ const isElectron = ( () => {
 	return () => foundElectron;
 } )();
 
+const isElectronMac = () => {
+	return isElectron() &&
+		get( window, 'process.platform', '' ) === 'darwin';
+}
```

Comment:
I hope that you tested this since I didn't. `process` is usually set by node and I suppose that Electron itself is the source when running there.